Packs/Coralogix/Integrations/Coralogix_unified.yml

```diff
@@ -0,0 +1,575 @@
+category: Data Enrichment & Threat Intelligence
```

Comment:
This file should be automatically removed after you will re-organize the pack structure